lib/matplotlib/axes/_axes.py

```diff
@@ -4832,9 +4832,7 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
 
         corners = ((xmin, ymin), (xmax, ymax))
         self.update_datalim(corners)
-        collection.sticky_edges.x[:] = [xmin, xmax]
```

Comment:
I don't understand what change caused this to _start_ working.

Reply:
To be honest I don't know either.